src/main/java/greencity/service/OpenHoursService.java

```diff
@@ -56,4 +57,19 @@
      * @return OpeningHours updated entity.
      */
     OpeningHours update(Long id, OpeningHours updatedHours);
+
+    /**
+     * Finds all {@code OpeningHours} records related to the specified {@code Place}.
+     *
+     * @param placeId to find by.
+     * @return a list of the {@code OpeningHours} for the place by id.
+     */
+    Set<OpeningHours> findAllByPlaceId(Long placeId);
+
+    /**
+     * Delete all {@code OpeningHours} records related to the specified {@code Place}.
+     *
+     * @param placeId to find by.
+     */
+    void deleteAllByPlaceId(Long placeId);
```

Comment:
Why it return nothing? We are making delete methods with anything in return statement, for example count of deleted rows.